### PR TITLE
added doux jésus

### PR DESCRIPTION
**Pull request by pelevesque**

### CODE REVIEW

File: lib/index.ts [hunk 1]
```diff
@@ -34,6 +34,7 @@ function getAllSwears(): Array<swearVariants> {
         [`charrue`],
         [`charogne`],
         [`gériboire`],
+        [`doux jésus`],
```
Comment (henrymyers) [inline lib/index.ts]: Great suggestion! Could you add it to the _jésus_ family though?

**Review by henrymyers — CHANGES_REQUESTED** [review]

File: lib/index.ts (hunk 1 above)
Comment (henrymyers) [reply lib/index.ts]: Also, we now capitalize names 😄

**Comment by pelevesque** [comment]
It should now be ready to merge.

**Review by henrymyers — APPROVED** [review]
Thanks!

**Pull request merged**